decoimpact/data/api/time_operation_type.py

```diff
@@ -24,3 +24,6 @@ class TimeOperationType(IntEnum):
     COUNT_PERIODS = 6
     MAX_DURATION_PERIODS = 7
     AVG_DURATION_PERIODS = 8
+    STDEV = 9
+    QUANT10 = 10
```

Comment:
same remark, can't these be combined?

Reply:
I have now combined these in an operation 'percentile(..)'